study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/task/AddStateOption.java

```diff
@@ -0,0 +1,85 @@
+package org.broadinstitute.ddp.studybuilder.task;
+
+import java.nio.file.Path;
+import java.util.List;
+
+import com.google.gson.Gson;
+import com.typesafe.config.Config;
+import org.broadinstitute.ddp.db.dao.JdbiActivityVersion;
+import org.broadinstitute.ddp.db.dao.PicklistQuestionDao;
+import org.broadinstitute.ddp.db.dto.ActivityVersionDto;
+import org.broadinstitute.ddp.exception.DDPException;
+import org.broadinstitute.ddp.model.activity.definition.question.PicklistOptionDef;
+import org.jdbi.v3.core.Handle;
+import org.jdbi.v3.sqlobject.SqlObject;
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AddStateOption implements CustomTask {
+
+    private static final Logger LOG = LoggerFactory.getLogger(AddStateOption.class);
+    private static final String[] studyGuids = {"cmi-pancan", "CMI-OSTEO", "cmi-brain", "rarex"};
```

Comment:
went through usage of state conf file (picklist-question-state-required.conf) and identified studies using the conf file.